tests/utils.go

```diff
@@ -272,7 +272,14 @@ func RunVMI(vmi *v1.VirtualMachineInstance, timeout int) *v1.VirtualMachineInsta
 
 func RunVMIAndExpectLaunch(vmi *v1.VirtualMachineInstance, timeout int) *v1.VirtualMachineInstance {
 	obj := RunVMI(vmi, timeout)
+	virtCli := kubevirt.Client()
+	kv := util2.GetCurrentKv(virtCli)
 	By(WaitingVMInstanceStart)
+	if kv.Spec.Configuration.EvictionStrategy != nil &&
```

Comment:
Can we check the original KV conf?

Reply:
Backport of https://github.com/kubevirt/kubevirt/pull/9948 will cover this case. Thanks